.github/workflows/ci.yaml

```diff
@@ -30,7 +34,14 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: Swatinem/rust-cache@v2
-      - run: cargo clippy -- -D warnings
```

Comment:
If we're now setting `-Dwarnings` in the env, do we need it here, too?

Reply:
Just tested it, and seems we do not need `-D warnings` here. Removed it.